distributed/dashboard/components/scheduler.py

```diff
@@ -3421,11 +3421,16 @@ def format(self, unit: str, val: Any) -> str:
     @without_property_validation
     @log_errors
     def update(self):
-        items = sorted(
-            (k, v)
-            for k, v in self.scheduler.cumulative_worker_metrics.items()
-            if isinstance(k, tuple)
-        )
+        items = defaultdict(float)
+        for k, v in self.scheduler.cumulative_worker_metrics.items():
+            if isinstance(k, tuple) and k[0] == "get-data":
+                items[k] = v
+            elif isinstance(k, tuple) and k[0] == "execute":
+                items[k[:1] + k[2:]] += v  # sum all span_id's together
+        # Note: this sort works because we removed span_id's. Otherwise, it would crash
+        # when the spans extension is disabled and span_id's are None as a consequence.
+        items = sorted(items.items())
```

Comment:
FYI @milesgranger 
Support in bokeh for the spans is highly desirable but I'd rather leave it to a different PR.